man/fread.Rd

```diff
@@ -163,53 +229,39 @@ cat("File size (MB):", round(file.info("test.csv")$size/1024^2),"\n")
 # 50 MB (1e6 rows x 6 columns)
 
 system.time(DF1 <-read.csv("test.csv",stringsAsFactors=FALSE))
-# 60 sec (first time in fresh R session)
+# 5.4 sec (first time in fresh R session)
```

Comment:
I would generally avoid posting timings in examples, despite putting into `dontrun`. As we can see, timings changes heavily over time, a 10 folds in below examples. It then make sense to keep them in NEWS.md where entries are bounds to a point in time. Moreover timings are highly affected by configuration, like number of threads, disk speed. It is not very practice to fleshout all such details in examples.